Dockerfile

```diff
@@ -5,6 +5,22 @@ MAINTAINER Juan Marin Otero <juan.marin.otero@gmail.com>
 RUN yum -y install epel-release
 RUN yum -y install nginx
 
+# Install npm
+RUN yum install -y npm
+
+# Install grunt-cli
+RUN npm install -g grunt-cli
+
+COPY . /
```

Comment:
Probably don't want to copy all of this into the root directory.  The pseudo-standard seems to be `/usr/src/app`.